packages/react/__tests__/src/components/Alert/index.js

```diff
@@ -7,16 +7,23 @@ const defaults = { show: false, heading: { text: 'hi' } };
 
 test('returns null if passed a falsey "show" prop', () => {
   const alert = mount(<Alert {...defaults}>{'hello'}</Alert>);
-  console.log(alert.debug())
```

Comment:
a log was accidentally checked in. The rest of the diff in this file is a result of prettier not being run on this file 😦